components/kyma-environment-broker/go.mod

```diff
@@ -46,6 +46,7 @@ require (
 )
 
 replace (
+	github.com/kyma-project/control-plane/components/provisioner => github.com/ksputo/control-plane/components/provisioner v0.0.0-20210611065312-8a619ab5a5e2
```

Comment:
I think we should merge the provisioner PR before we merge this and then we can get rid of the replace